Mathlib/CategoryTheory/Bicategory/LocallyDiscrete.lean

```diff
@@ -70,64 +90,79 @@ theorem eq_of_hom {X Y : LocallyDiscrete C} {f g : X ⟶ Y} (η : f ⟶ g) : f =
 
 end LocallyDiscrete
 
-variable (C) [Category.{v} C]
+variable {C : Type u} [Category.{v} C]
```

Comment:
Untested, but it seems to me `C` was declared above.
```suggestion
variable (C)
variable [Category.{v} C]
```

Reply:
Yes thats true! Fixed :)